zkp/circuits/basetokens/anon_base.circom

```diff
@@ -18,7 +18,7 @@ pragma circom 2.2.2;
 include "../lib/check-positive.circom";
 include "../lib/check-hashes.circom";
 include "../lib/check-sum.circom";
-include "../node_modules/circomlib/circuits/babyjub.circom";
+include "../scripts/node_modules/circomlib/circuits/babyjub.circom";
```

Comment:
see the comment above, the `package.json` should still be in the top folder, so that the `node_modules` folder doesn't need to change location.